src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECK_STATUS = 'bookstore/src/redux/categories/CHECK_STATUS';
+
+export default function categories(state = [], action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] [Optional] Same here, It would be better if you used arrow syntax for the reducer function instead of regular syntax.

Reply:
Thank you again!